scripts/orama/get-documents.mjs

```diff
@@ -0,0 +1,71 @@
+import zlib from 'node:zlib';
+
+import { slug } from 'github-slugger';
+
+import { NEXT_DATA_URL } from '../../next.constants.mjs';
+
+const nextPageData = await fetch(`${NEXT_DATA_URL}/page-data`);
+const nextAPIPageData = await fetch(`${NEXT_DATA_URL}/api-data`);
+
+const pageData = await nextPageData.json();
+const apiData = await nextAPIPageData.json();
+
+function inflate(data) {
```

Comment:
Is a function really needed? I'd rathe that this line gets duplicated than useless functions

Reply:
I made a separate function for readability purposes, but I can remove it